environments/dap/dap-prod/terraform/dap.tf

```diff
@@ -0,0 +1,5 @@
+module dap {
+  source       = "../../base"
+  project_name = "broad-dsp-monster-dap-prod"
+  vault_prefix = "secret/dsde/monster/dev/dog-aging/redcap-tokens/automation"
```

Comment:
this is another point in favor of `vault_prefix` having a description - i didn't realize it was the path for the redcap token.

it also looks like the value is currently unused?

Reply:
🤔 yeah this seems redundant, will remove.